### DISCUSSION TITLE
Rust RPC Async 接口探索 | SF-Zhou's Blog

### DISCUSSION THREAD

**Discussion by giscus** [SF-Zhou/sf-zhou.github.io#213] [Announcements]
# Rust RPC Async 接口探索 | SF-Zhou's Blog

https://sf-zhou.github.io/rust/rust_rpc_async.html

**Comment by SF-Zhou** [comment]
根据 [Rust Project goals for 2024](https://blog.rust-lang.org/2024/08/12/Project-goals.html)，2024 Edition 将会支持 async trait 和异步闭包，解决 Send 约束传播的问题，完成的话 async 将会非常易用。

**Comment by cppkeyboardman** [comment]
好巧，我也叫SF，也搞了一个基本一样的东西
